common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -51,9 +61,11 @@ Double/Triple-Notch
 
 The software notch filters used are very "spikey" being relatively narrow but good at attenuation at their center. On larger copters the noise profile of the motors is quite dirty covering a broader range of frequencies than can be covered by a single notch filter. In order to address this situation it is possible to configure the harmonic notches as double or triple notches that gives a wider spread of significant attenuation. To utilize this feature set :ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` to "1" for double notches, to "16" for triple notches.
 
-.. note:: Each notch has some CPU cost so if you configure both dynamic harmonics and double/triple notches (:ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` set to 3 or 18) you will end up with many notches on your aircraft per IMU. On flight controllers with 3 IMUs, this totals 24 or more notches which is computationally significant and could impact operation. For example, with F4 cpus with one IMU, using :ref:`INS_GYRO_RATE<INS_GYRO_RATE>` =0 (1khz) this is safe, as is 3 IMUs running with fast sampling (:ref:`INS_GYRO_RATE<INS_GYRO_RATE>` =1 (2khz) on H7 cpus. Use triple notches with caution to computational loading.
+.. note:: double notch option is no longer recommended since the triple notch option has been added.With a double notch, the maximum attenuation is either side of the center frequency, so on smaller aircraft with a very pronounced peak their use is usually counter productive.
+
+.. note:: Each notch has some CPU cost so if you configure multiple notches you can end up with many notches on your aircraft.For example, triple notches, using ESC telemetry will result in 3 notches per motor or 12 total notches. For example, with F4 cpus this should be acceptable, but enabling a second group of triple notches with :ref:`INS_HNTC2_ENABLE<INS_HNTC2_ENABLE>` could cause problems.
```

Comment:
one of the changes in 4.3 is the ability to have harmonics on notch-per-motor - you might want to mention this somewhere

Reply:
see line 44, is this what you had in mind?

Reply:
no, not the same. now if you set HMNCS you will get that number of harmonics on *each* motor. It use to be that you only got a single harmonic. So its possible to get a very large number of notches this way. For instance quad with triple notch and 3 harmonics == 4 x 3 x 3 == 36 notches

Reply:
added notes in mode and triple notch section